client/src/components/common/Header/style.css

```diff
@@ -0,0 +1,59 @@
+.header {
+  display: flex;
+  flex-direction: column;
+}
+
+.header_logo {
+  height: 2.5rem;
+  width: 6rem;
+  margin-left: 1rem;
+  margin-bottom: 60px;
+  text-align: center;
+}
+
+.header_logo-img {
+  height: 4.5rem;
+  width: 4rem;
+}
+.header_logo-title {
+  color: #3b5a6f;
+  font-size: 0.8rem;
+}
+
+.header_title {
+  font-size: 1.7rem;
+  color: #3b5a6f;
+  margin: 0;
+  margin-top: 0.6rem;
+  text-align: center;
+}
+
+.header_subTitle {
+  color: #ff7d00;
+}
+
+.sub-header {
+  color: #3b5a6f;
+  margin: 0;
+  padding: 0 1rem;
+  text-align: center;
+  font-size: 1.1rem;
+}
+
+@media only screen and (max-width: 600px) {
+  .header_logo {
+    align-self: center;
+    margin: 0.25rem 0;
+    margin-bottom: 30px;
+  }
+  .header_logo-img {
+    height: 2.5rem;
+    width: 3rem;
+  }
+  .header_title{
+    font-size: 1.3rem;
+  }
+  .sub-header{
+    font-size: 1.1rem !important;
```

Comment:
It's sometimes necessary but bear in mind that using `!important` should be the last resort - it can cause you hard to debug problems in the future.